packages/data/src/lib/dataview/index.ts

```diff
@@ -1 +1,2 @@
 export { CategoryDataView } from './category/category';
+export { CategorySourceDataView } from './category/CategorySourceDataView';
```

Comment:
Ok, when we mix them both together we should probably rename 'category/category' to 'category/Category', if we have inside a class